src/main/java/cms/gov/madie/measure/utils/GroupPopulationUtil.java

```diff
@@ -1,227 +1,69 @@
 package cms.gov.madie.measure.utils;
 
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import org.apache.commons.collections4.CollectionUtils;
-import org.bson.types.ObjectId;
-
-import org.apache.commons.lang3.StringUtils;
 
+import cms.gov.madie.measure.exceptions.GroupPopulationDisplayIdException;
 import gov.cms.madie.models.measure.Group;
-import gov.cms.madie.models.measure.MeasureObservation;
-import gov.cms.madie.models.measure.MeasureScoring;
+import gov.cms.madie.models.measure.Measure;
 import gov.cms.madie.models.measure.Population;
 import gov.cms.madie.models.measure.PopulationType;
-import gov.cms.madie.models.measure.Stratification;
 
 public class GroupPopulationUtil {
   private GroupPopulationUtil() {}
 
-  public static boolean areGroupsAndPopulationsMatching(
-      List<Group> originalGroups, List<Group> newGroups) {
-    boolean match = false;
-    if (!CollectionUtils.isEmpty(originalGroups) && !CollectionUtils.isEmpty(newGroups)) {
-      List<Group> matchedGroups =
-          originalGroups.stream()
-              .filter(
-                  originalGroup ->
-                      newGroups.stream()
-                          .allMatch(
-                              newGroup -> isGroupPopulationsMatching(originalGroup, newGroup)))
-              .toList();
-      match = CollectionUtils.isNotEmpty(matchedGroups);
-    }
-    return match;
-  }
-
-  public static boolean isGroupPopulationsMatching(Group originalGroup, Group newGroup) {
-    if (!StringUtils.equals(originalGroup.getScoring(), newGroup.getScoring())) {
-      return false;
-    }
-
-    if (!StringUtils.equals(originalGroup.getPopulationBasis(), newGroup.getPopulationBasis())) {
-      return false;
-    }
-
-    List<Population> originalValidPops = getValidPopulations(originalGroup);
-    List<Population> newValidPops = getValidPopulations(newGroup);
-    boolean populationsMatch = isPopulationMatch(originalValidPops, newValidPops);
-    if (!populationsMatch) {
-      return false;
-    }
+  public static void validatePopulations(Measure measure, Group group) {
+    String existingGroupDisplayId = group.getDisplayId();
+    String newGroupDisplayId = String.valueOf(getGroupNumber(group, measure.getGroups()));
 
-    List<MeasureObservation> originalValidObs = getValidObservations(originalGroup);
-    List<MeasureObservation> newValidObs = getValidObservations(newGroup);
-    boolean observationsMatch = isMeasureObservationMatch(originalValidObs, newValidObs);
-    if (!observationsMatch) {
-      return false;
+    if (existingGroupDisplayId != null
+        && !existingGroupDisplayId.equalsIgnoreCase("Group_" + String.valueOf(newGroupDisplayId))) {
```

Comment:
newGroupDisplayId is already a String

```suggestion
        && !existingGroupDisplayId.equalsIgnoreCase("Group_" + newGroupDisplayId)) {
```